### ISSUE TITLE
Remove staking address hack

### ISSUE THREAD

**Issue by samuelWilliams99** [mlabs-haskell/CardStarter-bulk-airdrop-script#25]
Once #19 is merged, we'll have the updated TxContraints, which include the staking address for pubkey outputs!
Previously, we had to create and pass around a map from pubkeyhash to full addresses in order to ensure we paid to the full address.
Now, we get this information in the Tx itself, so we can lose the map!

**Comment by nazrhom** [comment]
@samuelWilliams99 does it mean we can loose https://github.com/mlabs-haskell/CardStarter-bulk-airdrop-script/blob/staging/src/FakePAB/Constraints.hs#L53-L60 completely if we construct the constraints via `mustPayToPubKeyAddress`?

**Comment by samuelWilliams99** [comment]
Yep! Along with that `Map PubKeyHash PubKeyAddress` that we pass around :)
